docs/Tutorials/Orientation.md

```diff
@@ -13,14 +13,14 @@ as the logical directions \f$\xi\f$, \f$\eta\f$, and \f$\zeta\f$, where
 domain with multiple elements, the logical directions are not necessarily
 aligned on the interfaces between two elements, as shown in the figure below.
 As fluxes need to be communicated across the boundaries of adjacent elements,
-there needs to be a system in place that organizes the relative orientations of
-elements with respect to their neighbors. This is handled by OrientationMap.
+there needs to be a system in place that catalogs the relative orientations of
```

Comment:
As certain operations, (e.g. fluxes, limiting, ...) communicate information across the boundaries of adjacent elements, there needs to be a class that takes into account the relative orientations of neighboring elements.   This class is OrientationMap.